src/open_inwoner/templates/export/plans/plan_export.html

```diff
@@ -43,6 +43,6 @@ <h2 class="h2 h2--extra">{% trans "Bestanden" %}</h2>
     {% endif %}
 
     <h2 class="h2 h2--extra">{% trans "Acties" %}</h2>
-    {% include "export/include/actions.html" with actions=object.actions.all %}
+    {% include "export/include/actions.html" with actions=object.actions.visible %}
```

Comment:
This needs update in the profile as well. We also show the number of open actions in the this page and export them there.

Reply:
Good catch! I found the places in the profile and will update those as well.